osf/models/registration_bulk_upload.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from django.db import models
+
+from osf.models.base import BaseModel
+from osf.utils.fields import NonNaiveDateTimeField
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationBulkUpload(BaseModel):
```

Comment:
I would rename this `RegistrationBulkArchiveJob` as it's purpose is similar to `ArchiveJob` and have a model called `BulkUploadedRegistration` and `RegistrationBulkUpload` is a little confusing.

Reply:
I agree we need a better naming for the two classes. However, the "archiving" is not what the model and task do. Maybe `RegistrationBulkUploadJob` (replacing `RegistrationBulkUpload`). In addition, how about `BulkUploadedRegistrationCache` (replacing `BulkUploadedRegistration`).

Reply:
Good point there's no real archiving involved,  I like `RegistrationBulkUploadJob` the most but others are fine. Otherwise looking good.